.gitignore

```diff
@@ -6,6 +6,8 @@ coverage
 *.js.map
 *.d.ts
```

Comment:
Should we export the typings?

Reply:
Yes, but not on git. This is why we have the `.npmIgnore` file to override the `.gitIgnore`.